context/Alerts.tsx

```diff
@@ -36,6 +36,8 @@ export const AlertProvider: React.FC<{ children: React.ReactNode }> = ({ childre
       // Update the existing one
       setAlerts((curAlerts) => {
         const [prevAlert] = curAlerts.splice(idx, 1);
+        if (!prevAlert) return curAlerts;
```

Comment:
Prevents a runtime error when voting two times before the alert popup has disappeared